### DISCUSSION TITLE
android -secrets RENDEZVOUS_SERVER RS_PUB_KEY

### DISCUSSION THREAD

**Discussion by DarkMitsuhiro** [rustdesk/rustdesk#3136] [Q&A] [answered]
When creating releases through Nightly, the default server and public key are not registered in the apk.
I wrote Secrets with my settings according to the instructions https://rustdesk.com/docs/en/dev/build/all/
Please tell me what I did wrong.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/issues/3126
